tests/integration/test_vulnerability_detector/test_feeds/test_msu_inventory.py

```diff
@@ -0,0 +1,154 @@
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_msu_inventory.yaml')
+cases_path = os.path.join(TEST_CASES_PATH, 'cases_msu_inventory.yaml')
+
+# Test configurations
+configuration_parameters, configuration_metadata, case_ids = get_test_cases_data(cases_path)
+configurations = load_configuration_template(configurations_path, configuration_parameters, configuration_metadata)
+
+# Variables
+# This patches are searched for to verify they are added correctly from the Catalog even if not associated to a CVE
+patch_references = ['4465477', '5003711', '4470788']
+
+
+@pytest.mark.tier(level=2)
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=case_ids)
+def test_msu_catalog_patches(configuration, metadata, set_wazuh_configuration_vdt, truncate_monitored_files,
+                             clean_cve_tables_func, restart_modulesd_function):
+    '''
+    description: Check that patch information is added to the feed from the catalog for patches not directly related
+                 to a Vulnerability.
+
+    test_phases:
+        Setup:
+            - Set a custom Wazuh configuration.
+            - Restart wazuh-modulesd.
+        Test:
+            - Check in log that the database provider has been updated successfully.
+            - Query the DB to check patch is found in MSU_SUPERSEDENSE table
+            - Query the DB to check patch is not found in MSU table (Has not Vulnerability linked to it).
+        Teardown:
+            - Clean the database.
+            - Stop wazuh-modulesd.
+
+    wazuh_min_version: 4.5.0
+
+    tier: 2
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Wazuh configuration data. Needed for set_wazuh_configuration fixture.
+        - metadata:
+            type: dict
+            brief: Wazuh configuration metadata
+        - set_wazuh_configuration_vdt:
+            type: fixture
+            brief: Set the wazuh configuration according to the configuration data.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - clean_cve_tables_func:
+            type: fixture
+            brief: Clean all the CVE tables before and after running the test.
+        - restart_modulesd_function:
+            type: fixture
+            brief: Restart the wazuh-modulesd daemon.
+
+    assertions:
+        - Check that the feed is downloaded successfully.
+        - Check that the referenced patch is found in MSU_SUPERSEDENSE table
+        - Check that the referenced patch is not foundin MSU table (Has not Vulnerability linked to it).
```

Comment:
```suggestion
        - Check that the referenced patch is not foundin MSU table (has not Vulnerability linked to it).
```

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/4009/commits/2c11167dfbd10f2e4f190449c19831b52cde646a